_layouts/default.html

```diff
@@ -53,13 +53,13 @@ <h3 id="countdown">A gathering of Belfast cocoaheads</h3>
           
           <!-- Next meetup details starts  -->
           
-          <p>Next meetup: December 15th 2014 at 6:00pm</p>
+          <p>Next meetup: June 8th 2015 at 6:00pm</p>
```

Comment:
We should maybe say 5:45pm - keynote is at 6pm so would be good to get people in and settled.